### PR TITLE
Cmake cleanup

### PR DESCRIPTION
**Pull request by juan-lunarg**
I spread this into several atomic commits. Worth noting for review purposes.

### CODE REVIEW

File: cube/CMakeLists.txt [hunk 1]
```diff
@@ -49,7 +49,7 @@ else()
 endif()
 
 if(UNIX AND NOT APPLE) # i.e. Linux
-    include(FindPkgConfig)
+    find_package(PkgConfig)
```
Comment (charles-lunarg) [inline cube/CMakeLists.txt:52]: Why have find_package here and not in a more common place? (it looks like it is present here but not in the vulkaninfo CMakeLists.

**Review by charles-lunarg — APPROVED** [review]
Just a comment, otherwise the changes look good.

File: cube/CMakeLists.txt (hunk 1 above)
Comment (juan-lunarg) [reply cube/CMakeLists.txt:52]: Because I am error.

Comment (juan-lunarg) [reply cube/CMakeLists.txt:52]: I thought cube/CMakeLists.txt was the top level CMakeLists.txt not sure why

Comment (juan-lunarg) [reply cube/CMakeLists.txt:52]: Fixed now.

**Pull request merged**